app/presenters/sufia/select_type_presenter.rb

```diff
@@ -0,0 +1,35 @@
+module Sufia
+  class SelectTypePresenter
+    def initialize(concern)
+      @concern = concern
+    end
+
+    attr_reader :concern
+
+    def icon_class
+      translate('icon_class')
```

Comment:
@jcoyne I wonder if this should be the default thumbnail for the work and not a translated icon.